modules/tinymce/src/core/main/ts/content/PrePostProcess.ts

```diff
@@ -30,6 +30,12 @@ const withSerializedContent = <R extends EditorEvent<{ content: string }>>(conte
   }
 };
 
+const makeParserSettings = (editor: Editor): Partial<DomParserSettings> => ({
```

Comment:
I don't think we need to use partial here since the interface it already partial since all options are optional.